config/routes.rb

```diff
@@ -288,7 +288,7 @@
   # This is a legacy wild controller route that's not recommended for RESTful applications.
   # Note: This route will make all actions in every controller accessible via GET requests.
 
-  # match ':controller(/:action(/:id))(.:format)'
+  match ':controller(/:action(/:id))(.:format)'
```

Comment:
I've already made that change for PR #652 for the exact same reasons.  there shouldn't be a conflict, but I will verify.